introduction.md

```diff
@@ -3,3 +3,9 @@ Introduction
 
 Kenga learning Github
 I like Electives. 
+
+Deadly Skills:
+* Git
+* Java
+* Python
+* JS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?